glhe/aggregation/base_agg.py

```diff
@@ -1,36 +1,36 @@
-import os
```

Comment:
There are a lot of places where `os` isn't need anymore as I've moved to using `pathlib`.